bread/views/browse.py

```diff
@@ -1,4 +1,4 @@
-from typing import Callable, List, NamedTuple, Optional, Union
+from typing import Callable, List, NamedTuple, Optional, Tuple, Union
```

Comment:
This is a workaround because Python < 3.9 does not allow to subscribe to ```tuple```. Can be reverted around 2023 ;)